automated install/basic-install.sh

```diff
@@ -1845,6 +1845,7 @@ finalExports() {
     addOrEditKeyValPair "${setupVars}" "DNS_FQDN_REQUIRED" "${DNS_FQDN_REQUIRED:-true}"
     addOrEditKeyValPair "${setupVars}" "DNS_BOGUS_PRIV" "${DNS_BOGUS_PRIV:-true}"
     addOrEditKeyValPair "${setupVars}" "DNSMASQ_LISTENING" "${DNSMASQ_LISTENING:-local}"
+    addOrEditKeyValPair "${setupVars}" "GRAVITY_TMPDIR" "${GRAVITY_TMPDIR:-/tmp}"
```

Comment:
Don't think we really need this here, this function is meant to update `setupVars.conf` configuration lines if the values they point to were changed by the installation script.

Reply:
All good, removed that :)